### ISSUE TITLE
Babel transform so this really is for all

### ISSUE THREAD

**Issue by antiBaconMachine** [allouis/ivanka#5]
not all browsers support `let` so can we add a babel transform so this is really `4 ALL`?

**Comment by chrisnewtn** [comment]
Possible duplicate of #4

**Comment by antiBaconMachine** [comment]
No, this is more complete, more 4 ALL

**Comment by robstanley** [comment]
Yeah, I think https://github.com/allouis/ivanka/issues/4 is a subset of this. Maybe close that one and just use this one?

**Comment by chrisnewtn** [comment]
Don't listen to them @allouis, this ticket is too vague!
